app/controllers/stories_controller.rb

```diff
@@ -384,7 +386,7 @@ def story_params
     p = params.require(:story).permit(
       :title, :url, :description, :moderation_reason, :seen_previous,
       :merge_story_short_id, :is_unavailable, :user_is_author, :user_is_following,
-      :tags_a => [],
+      :repost_description, :tags_a => []
```

Comment:
This should go down on line 395 with the other mod-only attributes. Then you don't need the `story_params.except` above.

Reply:
Oh, I've not found it before. Thanks.

Reply:
You have to remove it from here or regular users can set the param.

Reply:
Sorry, but I think they can't do that, as this param excepts on line 395 when user is regular with other moderation params.